marshmallow_recipe/metadata.py

```diff
@@ -1,16 +1,17 @@
 import collections.abc
-from typing import Any, TypeGuard, final
+from typing import Any, Hashable, TypeGuard, final
 
 from .missing import MISSING
 from .validation import ValidationFunc
 
 
 @final
-class Metadata(collections.abc.Mapping[str, Any]):
-    __slots__ = ("__values",)
+class Metadata(collections.abc.Mapping[str, Any], Hashable):
```

Comment:
both `Mapping` and `Hashable` can be improted from both `collections.abc` and `typing` but why we use different approach for this types in this code? Why not to use import from `typing` for both?

<img width="654" alt="image" src="https://github.com/user-attachments/assets/95e9155f-c680-4c12-9c2a-a031df030566" />

Reply:
Done